l10n_br_account/models/account_invoice.py

```diff
@@ -128,6 +217,7 @@ def write(self, values):
         'date_invoice',
         'type')
     def _compute_amount(self):
+        self.fiscal_document_id._compute_amount()
         for inv_line in self.invoice_line_ids:
             if inv_line.cfop_id:
```

Comment:
I created this logic using two ifs because there are invoice without CFOP like service invoices or an account invoice without fiscal document (only linked with dummy fiscal document)